processing/src/main/java/io/druid/segment/LongDimensionIndexer.java

```diff
@@ -94,13 +99,27 @@ public DimensionSelector makeDimensionSelector(
     final int dimIndex = desc.getIndex();
     class IndexerLongColumnSelector implements LongColumnSelector
     {
+
+      @Override
+      public boolean isNull()
+      {
+        if (NullHandling.replaceWithDefault()) {
```

Comment:
Let's define clearer, should isNull() return depend on the current NullHandling configuration, or not. Because in some other places, at least in NilColumnSelector, it doesn't depend. Please update javadocs and review all impls.

Reply:
isNull need not depend on NullHandling configuration, removed the if block.